packaging/docker/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM centos:7
```

Comment:
We are now supporting centos:8

Reply:
I recall trying, but it was like it didn't exist a few days ago? Or at least I couldn't find it.

Reply:
Yeah there was a period where I knew that Trey was working with 8, but I couldn't seem to find it for Vagrant or Docker. Seems to be working now.

Reply:
is it worth having 2 images: `packaging/docker/el7/Dockerfile` `packaging/docker/el8/Dockerfile`